memtable.cc

```diff
@@ -211,8 +212,14 @@ memtable::find_or_create_partition(const dht::decorated_key& key) {
         memtable_entry* entry = current_allocator().construct<memtable_entry>(
             _schema, dht::decorated_key(key), mutation_partition(_schema));
         partitions.insert_before(i, *entry);
+        if (_table_stats) {
+            ++_table_stats->partition_creations;
+        }
         return entry->partition();
     } else {
+        if (_table_stats) {
+            ++_table_stats->partition_reuses;
+        }
```

Comment:
We already use the terms "partition_insertions" and "partition_hits" in the cache. Perhaps it's better to be consistent here in naming.

Reply:
Changed names, now using names like "memtable_partition_writes" and "memtable_partition_hits".